readme.md

```diff
@@ -56,26 +56,28 @@ Note: You're not restricted to digital ocean. Any machine with docker should be
 
        $ git clone https://github.com/speckleworks/SpeckleServer.git
        $ cd SpeckleServer
-       $ npm install
+       $ sudo npm install
```

Comment:
Theoretically, doing `sudo` on `npm install` is bad, especially in production. Was it really needed?

```suggestion
       $ npm install
```

We could add "when getting permission denied errors, preface your commands with `sudo` as generic help item.  [Obligatory reference](https://xkcd.com/149/)

Reply:
Did not know sudo was bad in this case. It was needed in my case.